### PR TITLE
docs(README): add instructions for debugging

### PR DESCRIPTION
**Pull request by algomaster99**
Signed-off-by: Aman Sharma <mannu.poski10@gmail.com>

@DianaLease or @irmerk , could anyone of you please add instructions for debugging on Mac?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ environment.
...
+   and see the console's output.
+
+If you are on the desktop version,
+
+-  **Windows:** You will need to
+   [download the developer tool](https://www.microsoft.com/en-us/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot:overviewtab)
+   separately. Once you are able to start the server and sideload the add-in,
+   you will see "Accord Project" as an option in the tool.
+   ![edge developer tool](readme_assets/devtool.png)
+
```
Comment (jolanglinais) [inline README.md]: A few suggestions: 

```
#### Debugging Add-In
- Web version: ...
- Desktop version:
  - **Windows**: ...
  - **MacOS**: ...
```

Also avoid "simply" in the spirit of [inclusive documentation](https://developers.google.com/style/inclusive-documentation)

**Comment by jolanglinais** [comment]
I'm not finding much useful information on a developer tool path aside from the built in Visual Basic one in Word, @DianaLease?

**Comment by DianaLease** [comment]
@irmerk @algomaster99 This [link](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/debug-office-add-ins-on-ipad-and-mac) worked for me to use the Safari console for debugging.

**Comment by jolanglinais** [comment]
@DianaLease maybe you could add the Mac specific content? I can't get that guide to work.

**Comment by DianaLease** [comment]
> @DianaLease maybe you could add the Mac specific content? I can't get that guide to work.

Sure, let's merge this and then I'll put in a separate PR

**Review by DianaLease — APPROVED** [review]

**Pull request merged**